docclean/cycle_gan.py

```diff
@@ -0,0 +1,147 @@
+import tensorflow as tf
+import tqdm
+from tensorflow_examples.models.pix2pix import pix2pix
+
+
+class CycleGan:
+    """
+    Cycle GAN in tensorflow based on the pip2pix.
+
+    Args:
+
+        checkpoint_path: Where to put the checkpoints
+
+        restor_checpoint: Restore old checkpoints or not
```

Comment:
Nitpick: maybe spell it `restore_checkpoint` ?

Reply:
Aah it's a spelling error, on it.